src/test/java/org/takes/tk/TkProxyTest.java

```diff
@@ -46,24 +47,70 @@
  * @todo #377:30min/DEV We need more tests for TkProxy.
  *  The tests should verify the different HTTP methods (GET, POST, etc),
  *  as well as the different combinations of request/response headers.
+ *
+ * @todo #458:30min We need to support the http method CONNECT.
+ *  Currently, the use of Connect request to  throws an IOException .
+ *  The com.jcabi.http.request.JdkReques classes send method is failing.
  */
 public final class TkProxyTest {
 
     /**
-     * TkProxy can work.
+     * An array of http methods for testing.
+     */
+    private static final String[] METHODS = {
+        RqMethod.POST,
+        RqMethod.GET,
+        RqMethod.PUT,
+        RqMethod.DELETE,
+        RqMethod.TRACE,
+    };
+
+    /**
+     * The expected result for the Head http request.
+     */
+    private static final String RESULT = "";
+
+    /**
+     * TkProxy can just work.
      * @throws Exception If some problem inside
      */
+    @SuppressWarnings("PMD.AvoidInstantiatingObjectsInLoops")
     @Test
     public void justWorks() throws Exception {
-        new FtRemote(new TkFixed("hello, world!")).exec(
+        for (final String method:TkProxyTest.METHODS) {
+            new FtRemote(new TkFixed("hello, world!")).exec(
+                new FtRemote.Script() {
+                    @Override
+                    public void exec(final URI home) throws IOException {
+                        MatcherAssert.assertThat(
+                            new RsPrint(
+                                new TkProxy(home).act(new RqFake(method))
+                            ).print(),
+                            Matchers.containsString("hello")
+                        );
+                    }
+                }
+            );
+        }
+    }
+
+    /**
+     * TkProxy can just work on Head.
+     * @throws Exception If some problem inside
+     */
+    @Test
+    public void justWorksOnHEAD() throws Exception {
+        new FtRemote(new TkFixed("Head test!")).exec(
             new FtRemote.Script() {
                 @Override
                 public void exec(final URI home) throws IOException {
                     MatcherAssert.assertThat(
                         new RsPrint(
-                            new TkProxy(home).act(new RqFake("PUT"))
+                            new TkProxy(home).act(
+                                new RqFake(RqMethod.HEAD)
+                            )
                         ).print(),
-                        Matchers.containsString("hello")
+                        Matchers.containsString(TkProxyTest.RESULT)
```

Comment:
@sebing I didn't get what you check here ?
I guess any string contains `""`, isn't it?

Anyway I think the existing methods enough for the test, let's remove test for  `HEAD`